src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -202,10 +202,17 @@ def load_arguments(self, _):
         c.argument('certificate_password', options_list=['--password', '-p'], help='The certificate file password')
         c.argument('prompt', options_list=['--show-prompt'], action='store_true', help='Show prompt to upload an existing certificate.')
 
+    with self.argument_context('containerapp env certificate create') as c:
+        c.argument('hostname', options_list=['--hostname'], help='The custom domain name.')
```

Comment:
It is not necessary to specify the options_list when there is only one value in options_list and it is the same as the parameter name. 
```suggestion
        c.argument('hostname', help='The custom domain name.')
```

Reply:
Hi @yanzhudd 

The `hostname` has been defined in command `az containerapp env`:
```
    with self.argument_context('containerapp env', arg_group='Custom Domain') as c:
        c.argument('hostname', options_list=['--custom-domain-dns-suffix', '--dns-suffix'], help='The DNS suffix for the environment\'s custom domain.')
```

Only with `options_list=['--hostname'], ` it can show as expected.